Manifest.toml

```diff
@@ -286,9 +286,11 @@ version = "1.3.0"
 
 [[deps.RegistryTools]]
 deps = ["AutoHashEquals", "LibGit2", "Pkg", "SHA", "UUIDs"]
-git-tree-sha1 = "aceb02a844394332608c88d7da6217daf2b5e4e5"
+git-tree-sha1 = "ef658f3c70d6d3b75f75c8f5116d3bc08f5b9243"
+repo-rev = "v1.9.0"
+repo-url = "https://github.com/JuliaRegistries/RegistryTools.jl.git"
```

Comment:
Is there a reason this is using a URL + tag instead of a normally registered version?